student.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+
+# inherite Person class
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom)
```

Comment:
- [ ] Your initialize method does not call the super method with all the required methods, please kindly add the `name, age, and parent_permission` parameters.

eg: 

``` ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```